docs/extensions/storage-transformers/sharding/v1.0.rst

```diff
@@ -35,6 +35,15 @@ Sharding co-locates multiple chunks within a storage object, bundling them in
 shards.
 
 
+Status of this document
+=======================
+
+.. warning::
+    This document is a draft for review and subject to changes.
+    It will become final when the `Zarr Enhancement Proposal (ZEP) 2 <https://zarr.dev/zeps/draft/ZEP0001.html>`_
```

Comment:
There's a typo in the hyperlink here. It should point to ZEP0002 at https://zarr.dev/zeps/draft/ZEP0002.html.

Reply:
Good catch, thanks :+1: If you want to, you can even propose specific changes in a comment using the suggestion button:
![Screenshot from 2022-11-22 10-07-25](https://user-images.githubusercontent.com/7216331/203272608-15270b0d-f3d0-453c-bce5-8bb38b968e44.png)

They appear like below, and I can simply accept them to this PR:

```suggestion
    It will become final when the `Zarr Enhancement Proposal (ZEP) 2 <https://zarr.dev/zeps/draft/ZEP0002.html>`_
```